universe/com.goal_smart/manifest.tt

```diff
@@ -75,100 +77,131 @@ class @com.goal_smart
   monitorable list query teamFixtures(in req team_id: Entity(com.goal_smart:soccer_team) #_[prompt="what team do you want out check out"],
                         out league : Entity(com.goal_smart:soccer_league)
                         #_[canonical = {
-                        base = ["league", "division"]
+                          base = ["league", "division"]
                         }],
                         out team1: Entity(com.goal_smart:soccer_team)
                         #_[canonical = {
-                        base = ["team one", "side one", "group one"]
+                          base = ["team one", "side one", "group one"]
                         }],
                         out team2: Entity(com.goal_smart:soccer_team)
                         #_[canonical = {
-                        base = ["team two", "side two", "group two"]
+                          base = ["team two", "side two", "group two"]
+                          verb = ["plays against #", "faces #", "faces against # ", "play # next, "played", "play against # next"]
                         }],
                         out venue: String
                         #_[canonical = {
-                        base = ["location", "place", "venue"]
+                          base = ["location", "place", "venue"]
                         }])
-  #_[canonical=["team fixtures", "team matches", "team games"]]
+  #_[canonical=["team fixtures", "team matches", "team games, "upcoming games"]]
```

Comment:
Syntax error here. You should make sure to run `make lint` to catch these errors before you submit.

Also, canonical forms should be singular whenever possible. Plural forms are added automatically. If you add a plural yourself, you might end up with ungrammatical sentences when the agent speaks.

Also, I think the basic canonical form should be "soccer game" or maybe "fixture", without the word team, but with an explicit reference to soccer, because that's what we're talking about. 
You should add relevant canonical forms to `team_id`, otherwise we won't generate the right sentences.
You can use `#_[canonical_short]` to write a shorter phrase where the domain is implicit (like "game"), to be used in follow-up commands in the same context.

There is also a lot of overlap between `teamFixtures` and `previousFixtures`, and maybe we should consolidate the two functions in some way, make it just a database of "games" with a date, and you can check previous as `date <= $now`.